addons/account/views/report_invoice.xml

```diff
@@ -123,6 +124,12 @@
 
                                     <tr t-att-class="'bg-200 fw-bold o_line_section' if line.display_type == 'line_section' else 'fst-italic o_line_note' if line.display_type == 'line_note' else ''">
                                         <t t-if="line.display_type == 'product'" name="account_invoice_line_accountable">
+                                            <td name="td_img" class="td-img" style="max-width: 100px;">
+                                                <img t-if="line.product_id.image_128"
+                                                    class="img o_image_64_max"
+                                                    style="max-width: 64px; max-height: 64px;"
+                                                    t-att-src="image_data_uri(line.product_id.image_128)"/>
+                                            </td>
```

Comment:
Note that this diff (same in `sale` module) is just dummy data for testing purpose, as hinter in the commit which introduces it.
![image](https://github.com/odoo/odoo/assets/30048408/0a458192-0609-4ad0-87b5-638c2ed28902)

This will be removed before merge.